### ISSUE TITLE
Version 4.24.4 numbers don't match

### ISSUE THREAD

**Issue by electrocucaracha** [mikefarah/yq#1180]
**Describe the bug**
The `4.24.4` binary is displaying a wrong version number

**Input Yaml**
NA

**Command**
```bash
wget https://github.com/mikefarah/yq/releases/download/v4.24.4/yq_linux_386
chmod +x yq_linux_386
```

**Actual behavior**

```bash
$ ./yq_linux_386 --version
yq (https://github.com/mikefarah/yq/) version 4.24.2
```

**Expected behavior**

```yaml
$ ./yq_linux_386 --version
yq (https://github.com/mikefarah/yq/) version 4.24.4
```

**Additional context**
It seems like the [version](https://github.com/mikefarah/yq/blob/v4.24.4/cmd/version.go#L14) number hasn't been updated properly.

**Comment by mikefarah** [comment]
Yep my bad :(  - the Github Action I was using to release stopped working, and I had to release multiple times to get it to work. I forgot to update the embedded version. Will fix in the next release.

**Comment by mikefarah** [comment]
Fixed in 4.24.5
